nss-day-cohort-46/rare-server-tragic-trolls
# Nc get single post by

#The server can now return a single post object if you end the route with posts/#

#To Test

```
ttb nc-getSinglePostById
```

Go to postman and type http://localhost:8088/posts/6 and see if it returns the sixth post back to you with the correct data on it.
